src/integration/pages/storyPage/user.canonical.js

```diff
@@ -0,0 +1,9 @@
+export default ({ imageAltText, imageCaptionText }) => {
```

Comment:
Same with this function here - can we have it defined in `/integration/common/user.canonical.js` and re-use it for each page type?